### PR TITLE
Update Orders

### PR DESCRIPTION
**Pull request by Drupi**
qweasdzxcasdqweasdzxcasdqwe

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class OrdersController < ApplicationController
+    
+def index; end
```
Comment (ozimeu) [inline app/controllers/orders_controller.rb]: inconsistent indentation

File: app/controllers/orders_controller.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+    def show
+        @order = Order.find(params[:id])
+    end
+end
```
Comment (ozimeu) [inline app/controllers/orders_controller.rb]: new line

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (SmaQu) [reply app/controllers/orders_controller.rb]: Inconsiste what?

File: app/views/orders/show.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<%= @order.order_name %>
```
Comment (ozimeu) [inline app/views/orders/show.erb]: new line

File: app/controllers/orders_controller.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class OrdersController < ApplicationController
+    
+    def index
+
```
Comment (ozimeu) [inline app/controllers/orders_controller.rb]: multiline empty method

**Pull request merged**